File: python/class 3/question 4.py
toonies_amount=int(input("How many toonies do you have? "))
loonies_amount=int(input("How many loonies do you have? "))
quarters_amount=int(input("How many quaters do you have? "))
dimes_amount=int(input("How many dimes do you have? "))
nickles_amount=int(input("How many nickles do you have? "))

toonies = 2*toonies_amount
loonies = 1*loonies_amount
quarters = 0.25*quarters_amount
dimes = 0.1*dimes_amount
nickles = 0.05*nickles_amount

total_cents=(quarters+dimes+nickles)
total_dollars=toonies+loonies
total=total_cents+total_dollars

rounding_total = f'{total:.2f}'

print(f"The total value of your coins comes out to: ${rounding_total}")

File: java/class 3/Question4.java
import java.util.Scanner;

public class Question4{
  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("How many toonies do you have? ");
    Integer toonies_amount = input.nextInt();
    System.out.print("How many loonies do you have? ");
    Integer loonies_amount = input.nextInt();
    System.out.print("How many quaters do you have? ");
    Integer quarters_amount = input.nextInt();
    System.out.print("How many dimes do you have? ");
    Integer dimes_amount = input.nextInt();
    System.out.print("How many nickles do you have? ");
    Integer nickles_amount = input.nextInt();
    input.close();

    Integer toonies = 2 * toonies_amount;
    Integer loonies = 1 * loonies_amount;
    double quarters = 0.25 * quarters_amount;
    double dimes = 0.1 * dimes_amount;
    double nickles = 0.05 * nickles_amount;
    
    double total_cents = quarters + dimes + nickles;
    Integer total_dollars = toonies + loonies;
    double total = total_cents + total_dollars;
    

    System.out.println("The total value of your coins comes out to: $" + String.format("%.2f", total));


  }

}